docs/doctoring/dependency-security-baseline.md

```
@@ -140,3 +140,9 @@ Python Packaging Authority. (2026c). *pypdf 6.15.0*. Python Package Index.
 
 Python Packaging Authority. (2026d). *setuptools 83.0.0*. Python Package Index.
     Retrieved August 4, 2026, from https://pypi.org/project/setuptools/83.0.0/
+
+
+### pypdf updates
```

markdownlint-cli2 (0.23.2)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)